packages/db/prisma/schema.prisma

```diff
@@ -52,11 +53,18 @@ model Activity {
 }
 
 model Absence {
-  userId      Int
-  user        User     @relation(fields: [userId], references: [id])
-  absenceDate DateTime @db.Date
+  participant   Participant @relation(fields: participantId, references: id)
+  participantId Int
+  absenceDate   DateTime    @db.Date
 
-  @@unique([absenceDate, userId])
+  @@unique([absenceDate, participantId])
+}
+
+model Participant {
+  id       Int       @id @default(autoincrement())
+  name     String
+  user     User[]
```

Comment:
🥲 last change I promise, but should this be users instead?

Reply:
for line 66

Reply:
good catch, thank you!